Makefile

```diff
@@ -1,42 +1,54 @@
 IMAGE=quay.io/packit/packit
-TESTS_IMAGE=packit-tests
+TEST_IMAGE=packit-tests
 
 CONTAINER_ENGINE ?= $(shell command -v podman 2> /dev/null || echo docker)
-TESTS_RECORDING_PATH=tests_recording
-TESTS_TARGET ?= ./tests/unit ./tests/integration ./tests/functional
+TEST_RECORDING_PATH=tests_recording
+TEST_TARGET ?= ./tests/unit ./tests/integration ./tests/functional
 CONTAINER_RUN_WITH_OPTS=$(CONTAINER_ENGINE) run --rm -ti -v $(CURDIR):/src --security-opt label=disable
 CONTAINER_TEST_COMMAND=bash -c "pip3 install .; make check"
+# These should work for Fedora, CentOS, RHEL...:
+PACKAGE_CHECKER ?= rpm -q --quiet
+PACKAGE_INSTALLER ?= dnf install
 
 # In case you don't want to use pre-built image
 image:
 	$(CONTAINER_ENGINE) build --rm --tag $(IMAGE) .
 
-tests_image:
-	$(CONTAINER_ENGINE) build --rm --tag $(TESTS_IMAGE) -f Dockerfile.tests .
+tests-image:
+	$(CONTAINER_ENGINE) build --rm --tag $(TEST_IMAGE) -f Dockerfile.tests .
 	sleep 2
```

Comment:
`sleep 2` could be removed if not `check-*` target depends on building.

Reply:
Removed sleep 2 as suggested.